pkg/kv/txn_coord_sender.go

```diff
@@ -1010,13 +1010,18 @@ func (tc *TxnCoordSender) tryAsyncAbort(ctx context.Context) {
 		tc.AnnotateCtx(context.Background()), "kv.TxnCoordSender: aborting txn", func(ctx context.Context) {
 			// Use the wrapped sender since the normal Sender does not allow
 			// clients to specify intents.
+			log.Infof(ctx, "sending end transaction request to abort")
```

Comment:
`VEventf` or if you ever want this in the shared log, pls add more info - the txn id or something

Reply:
Had some leftover debug statements. Removed.